src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -244,12 +244,10 @@ private SQLServerResultSet getResultSetFromInternalQueries(String catalog,
         SQLServerResultSet rs = null;
         try {
             rs = ((SQLServerStatement) connection.createStatement()).executeQueryInternal(query);
-
         }
         finally {
-            if (null != orgCat) {
+            if (null != orgCat)
```

Comment:
I don't think we ever put brackets when we could omit them. so i omit them here as well